keyboards/sirius/unigo66/unigo66.h

```diff
@@ -0,0 +1,105 @@
+#ifndef UNIGO66_H
+#define UNIGO66_H
+
+#include "quantum.h"
+
+#define ______ KC_TRNS
```

Comment:
QMK defines a keycode for this automatically.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

```suggestion
```

Hence the big update for Lines 33-48 of the default `keymap.c`.